### PR TITLE
Hello Rails-React

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
# NOTE FOR CODE REVIEWER ❗ 
- My stylelint checks are passing locally but on GitHub, they are not passing. If you have a clue why, please let me know. We are not required to use CSS in this project and I was wondering if stylelint is necessary?

# Project Details ⚙️ 
- Create a new Rails app called 'hello-rails-react'.
- Initialize the project with Git.
- Set up theproject with webpacker the react-rails gems 
- Postgres database set up.
- Create a table for storing messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from the table.
- Create a static view that will be the root of the app.
- Create the App component with react-router and render it in my static view.
- Create the Greeting component that will display a greeting.
- Create a store, an action and a reducer that will connect to my API endpoint to get the random greeting.
- Display the random greeting in my Greeting component.

### CODE REVIEW

**Comment by t-yanick** [comment]
Hello @Lucas-Masaba, please delete(`in your local repository`) the `node_modules directory`, `package-lock.json file` and then run `npm install` again on your local repository, commit and push the changes(`resolve conflicts if there are any`). That should solve the issue.

**Comment by Lucas-Masaba** [comment]
Thank you. Let me do that

File: app/javascript/components/HelloReactRails.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react"
+// import PropTypes from "prop-types"
```
Comment (t-yanick) [inline app/javascript/components/HelloReactRails.js:2]: - [OPTIONAL] Please kindly remove commented code from your code base, as it makes your code base dirty and less readable.

File: app/javascript/components/HelloReactRails.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <h1>Greeting</h1>
+      <div>Message: {message.message}</div>
+    </div>
+    );
+
+}
+
+// HelloReactRails.propTypes = {
+//   greeting: PropTypes.string
+// };
```
Comment (t-yanick) [inline app/javascript/components/HelloReactRails.js:27]: - [OPTIONAL] Please kindly remove commented code from your code base, as it makes your code base dirty and less readable.

**Comment by Lucas-Masaba** [comment]
That worked, thank you @t-yanick

**Review by t-yanick — APPROVED** [review]
Hi @Lucas-Masaba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/components/HelloReactRails.js (hunk 2 above)
Comment (t-yanick) [reply app/javascript/components/HelloReactRails.js:27]: You are welcome man.

**Comment by t-yanick** [comment]
> That worked, thank you @t-yanick

You are welcome man. Happy coding.

**Pull request merged**